Shops/household_shop/homeworks/hw_auth_service/app/routes.py

```diff
@@ -0,0 +1,73 @@
+from flask import jsonify, request
+from models import User
+from config import app, db, answer_by_code
+import hashlib
+import jwt
+
+
+def credentials_ctrl(): 
+    if not request.form.get('Email') or not request.form.get('Password'):
+        return answer_by_code.get("Credential")
+
+
+def get_md5_password(password):    
+    md5_hash = hashlib.md5()
+    md5_hash.update(bytes(password, 'utf-8'))
+    return md5_hash.hexdigest()
+
+
+def check_user_data_for_reg():
+    email = request.form.get('Email')
+    password = request.form.get('Password')
+    
+    if not User.query.get(email):
+        password = get_md5_password(password)
+
+        db.session.add(
+            User(email=email, password=password)
+        )
+        db.session.commit()
+
+        answer = answer_by_code.get(201)
+
+    else:
+        answer = answer_by_code.get(400).get("already_exists")
+
+    return answer
+
+
+def check_user_data_for_auth():
+    email = request.form.get('Email')
```

Comment:
Лучше через get_json получать данные, это более ростовский подход

Reply:
Согласен, но postman мне выкидывает ошибку: "Did not attempt to load JSON data because the request Content-Type was not &#39;application/json&#39;"

Прописал в Headers Content-Type: application/json, ошибка осталась, что ещё может быть?

Reply:
@mega-yujin, может ты подскажешь в чём может быть дело?

Reply:
Всем спасибо, вопрос снят)

Reply:
Ты не применил метод jsonify